src/ui/components/SecondaryToolbox/ReduxDevTools.tsx

```diff
@@ -75,35 +75,50 @@ export const ReduxDevToolsPanel = () => {
     const rootComponent = rootRef.current;
     const store = rootComponent.store!;
 
-    const matchingAnnotationsByTimeRange = reduxAnnotations.filter(annotation => {
-      return currentTimestamp != null && annotation.time < currentTimestamp;
+    const allInstanceIds: string[] = ["default"];
+
+    reduxAnnotations.forEach(annotation => {
+      annotation.action.request.instanceId;
```

Comment:
What's this line doing? Seems like an accident?

Reply:
Yep, shoulda removed that.